tests/Nhs.Appointments.Core.UnitTests/BookingServiceTests.cs

```diff
@@ -25,7 +25,7 @@ public BookingServiceTests()
                 TimeProvider.System);
         }
 
-        [Fact (Skip = "flakey test. needs investigation")]
+        [Fact]
```

Comment:
Maybe this test still wants to be skipped - I've just addressed the warnings about `task.Wait()` in a fairly naïve way but it's passed 5 times locally and twice on the pipeline, so that might've fixed the flakiness? Maybe? 🤷